src/screens/Ticketing/Purchase/Confirmation/index.tsx

```diff
@@ -34,6 +34,16 @@ import {useAuthState} from '@atb/auth';
 import {usePreviousPaymentMethod} from '../saved-payment-utils';
 import MessageBoxTexts from '@atb/translations/components/MessageBox';
 import {useFirestoreConfiguration} from '@atb/configuration/FirestoreConfigurationContext';
+import {
+  useHasEnabledMobileToken,
+  useMobileTokenContextState,
+} from '@atb/mobile-token/MobileTokenContext';
+import {
+  findInspectable,
+  getDeviceName,
+  isTravelCardToken,
+} from '@atb/mobile-token/utils';
+import {getOtherDeviceIsInspectableWarning} from '../../Ticket/utils';
```

Comment:
Not sure what import guidelines we are following, and the pros and cons of the possible options, but my personal feeling is that multiple navigation dots can be less readable than explicit references.

```suggestion
import {getOtherDeviceIsInspectableWarning} from '@atb/screens/Ticketing/Ticket/utils';
```

Reply:
I was actually considering the same thing. And I'm not sure what's best, but since both these files are part of screens/Ticketing, maybe this is fine as is? I dont think we have a policy on this, so I just stuck with whatever was auto generated ¯\_(ツ)_/¯

Reply:
(Turns out, shrugging is hard in markdown: `¯\\\_(ツ)\_/¯` -> ¯\\\_(ツ)\_/¯)

Reply:
I had a feeling they were auto generated 😄 

As long as you think it is the most readable way, it works just fine! 👌

Reply:
Maybe not more readable in this case, so I'm not against changing it this once, but it will be a lot more effort trying to enforce something other than what's auto generated in the future, so I'm leaning towards keeping it.